consumer/start-consumer.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# set image and container names
+IMAGE_NAME="kafka-consumer"
+CONTAINER_NAME="kafka-consumer-container"
+
+# Check if the Docker image already exists
+if [[ "$(docker images -q $IMAGE_NAME 2> /dev/null)" == "" ]]; then
```

Comment:
this option could be managed by a flag to allow to rebuild the image

Reply:
done